gradle.properties

```diff
@@ -31,8 +31,8 @@ gradleLombokPluginVersion=3.2.0
 gradleMkdocsPluginVersion=1.0.0
 gradleSshPluginVersion=2.9.0
 grpcVersion=1.17.1
-guavaVersion=28.2-jre
-hadoopVersion=2.8.5
+guavaVersion=25.0-jre
```

Comment:
We should not be going back to an older version of guava.

Reply:
Yes Andrei, there was no other choice. All  newer versions of guava do not agree with hadoopVersion 2.9.2 or later.

Reply:
@ravibeta as per https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-hdfs/3.1.2 > `Compile Dependencies` section, v3.1.2 depends on v`29.0-jre` of Guava. So, with the HDFS upgrade to this `3.1.2` you might be able to upgrade Guava now.

Reply:
Yes Ravi